config/hypershift-operator/release-info.json

```diff
@@ -0,0 +1,2268 @@
+{
```

Comment:
This file is no longer required — I removed it from gitignore and so you probably didn't mean to commit it